bedrock/foundation/templates/foundation/annualreport/2020/index.html

```diff
@@ -399,48 +399,81 @@ <h2>Privacy not included</h2>
+            <h2><span class="highlight green">Building a more trustworthy internet</span></h2>
+            <p>The future of the internet depends on being able to fix the problems we are facing today: untrustworthy algorithms, bias online, social media’s impact on society and democracies, an opaque business model that lends itself to harm. Yet to enact smart and comprehensive fixes that will help us build a better internet, we must first understand the root causes of the problems. </p>
+            <p>This requires transparency. And a lot more of it than we have today.</p>
+            <p>Our work this year helped shine a light on what’s really happening online through new research, new innovations and teaming up with new and old friends to push for change. This might not be surprising coming from a company founded on open-source, but we just think the internet is a better place when it’s more open and transparent, and we intend to do whatever we can to help fight for it.</p>
            </div>
         </div>
       </div>
 
 
-        <div class="has-modal" data-modal-id="third-section-card-1">
+      <div class="has-modal" data-modal-id="privacy-advertising">
         {% call split(
-          image_url='img/foundation/annualreport/2019/victory-ring.jpg',
-          include_highres_image=True,
+          image_url='img/foundation/annualreport/2020/firefox-privacy.jpg',
           block_class='mzp-l-split-body-narrow mzp-t-split-nospace'
          ) %}
-           <span>Nice tiny headline</span>
-           <h2>Lorem ipsum dolor sit.</h2>
-           <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nostrum odio sunt iste, nobis accusamus tenetur.</p>
-           <button type="button" class="mzp-c-button mzp-t-secondary">
-             This button does something
-           </button>
+           <span>Making a statement — in the New York Times</span>
+           <h2>Privacy Preserving Advertising series</h2>
           {% endcall %}
       </div>
 
         <div class="mzp-l-card-half mzp-l-content">
           {{ card(
-            title='This is a title',
-            ga_title='lThis is a title',
-            tag_label='Tiny title',
-            desc='This is a description of the card',
+            title='Regrets Reporter research',
+            ga_title='Regrets Reporter research',
+            tag_label='Holding YouTube accountable for harmful recommendations',
             class='has-modal',
-            attributes='data-modal-id="third-section-card-2"',
+            attributes='data-modal-id="youtube-regrets"',
             aspect_ratio='mzp-has-aspect-16-9',
-            image_url='img/foundation/annualreport/2019/misinfo-mondays.jpg',
+            link_url='privacy-control',
```

Comment:
Extra `link_url` here, but it seems like the second one supersedes the first so it's not causing an error. Still worth fixing.

Also since these are linking to the modal ID should they include the `#`? The JS seems to handle opening the modal without a hash but it's not a working link otherwise (can't open it in a new tab, for example).